### PR TITLE
New central settlement

### PR DESCRIPTION
**Pull request by bothadeon**
The DA changes has been implemented.
Additional unit tests added.
Swagger.json attached:
[swagger.json](https://raw.githubusercontent.com/bothadeon/central-settlement/07a716619bcf2b3b6b11303d7a8cd23823c75264/config/swagger.json)
 Notes from DA review:
[WarrenOneNotes.pdf](https://github.com/mojaloop/central-settlement/files/2240630/WarrenOneNotes.pdf)

### CODE REVIEW

**Comment by bothadeon** [comment]
Resolved merge conflicts

**Comment by bothadeon** [comment]
Oops !

Do another pull request?

Deon

**Comment by bothadeon** [comment]
Resolve merge conflicts for the new base branch : Develop

**Comment by bothadeon** [comment]
Resolve conflicting files

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-    "version": "1.0.0",
     "name": "central-settlement",
     "description": "Central settlements hosted by a scheme to record and settle settlements.",
+    "version": "1.0.0",
     "license": "Apache-2.0",
     "private": true,
     "author": "Valentin Genev <vgenev@gmail.com>",
```
Comment (mdebarros) [inline package.json]: Please remove the author tag.

File: tests/unit/settlementWindows/{id}.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
+'use strict';
```
Comment (mdebarros) [inline tests/unit/settlementWindows/{id}.js]: Missing license

File: data/settlements.js [hunk 3]
```diff
@@ -4,6 +4,82 @@ var Mockgen = require('./mockgen.js');
  * Operations on /settlements
```
Comment (mdebarros) [inline data/settlements.js]: Missing license?

File: handlers/settlements/{settlementId}/participants/{participantId}/accounts/{accountId}.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
+'use strict';
```
Comment (mdebarros) [inline handlers/settlements/{settlementId}/participants/{participantId}/accounts/{accountId}.js]: Missing license?

File: server.js [hunk 5]
```diff
@@ -12,7 +12,7 @@ const openAPIOptions = {
```
Comment (mdebarros) [inline server.js]: Missing license at the top?

File: swagger-158.json [hunk 6]
```diff
@@ -0,0 +1,944 @@
+{
```
Comment (mdebarros) [inline swagger-158.json]: I see not POST for /settlements?

**Comment by mdebarros** [comment]
+1 looks good.

**Pull request merged**